src/steamship/data/plugin_version.py

```diff
@@ -65,7 +63,7 @@ def create(
         is_public: bool = None,
         is_default: bool = None,
         config_template: Dict[str, Any] = None,
-    ) -> Response[PluginVersion]:
+    ) -> Task[PluginVersion]:
```

Comment:
Side comment: I kind of like that we're calling the response `Task` instead of `Response`. 

It feels like it explains the nature of what we're returning far better.

Reply:
I also like the as a user I know what I should do based on the response type. I could write this re-usable code snippet:

```
if isinstance(response, Task): 
  # wait and fetch
  response.wait()
  o = response.output
else: 
   o = response 
```